connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -522,17 +534,51 @@ ble_error_t Gap::connect(
     const ConnectionParameters &connectionParams
 )
 {
+    tr_info("Connect - "
+            "peerAddressType=%s, "
+            "peerAddress=%s, "
+            "_filterPolicy=%s, "
+            "_ownAddressType=%s, ",
+            to_string(peerAddressType),
+            to_string(peerAddress),
+            to_string(connectionParams.getFilter()),
+            to_string(connectionParams.getOwnAddressType()));
+
+    // Trace connection params for each enabled PHY
+    for (size_t i = 0; i < connectionParams.getPhySet().count(); ++i) {
```

Comment:
Could you add a trace in `on_unnexpected_error` ?

Reply:
Is that in a different source file?

Reply:
Same source file, wrong spelling from me: `on_unexpected_error`